hpeOneView/resources/search/labels.py

```diff
@@ -37,61 +37,10 @@ class Labels(object):
     URI = '/rest/labels'
     RESOURCES_PATH = '/resources'
 
-    def __init__(self, con):
-        self._connection = con
-        self._client = ResourceClient(con, self.URI)
+    def __init__(self, connection, data=None):
+        super(Labels, self).__init__(connection, data)
 
-    def get_all(self, start=0, count=-1, filter='', sort=''):
```

Comment:
you need to define get_all() method in this class because get_all() method in parent class supports only few filters whereas for this resource, we have "category, count, fields, namePrefix, sort, start, view" as filters.

Reply:
sure, its updated now.